tests/www/views/test_views_trigger_dag.py

```diff
@@ -214,3 +214,16 @@ def test_viewer_cant_trigger_dag(app):
         resp = client.get(url, follow_redirects=True)
         response_data = resp.data.decode()
         assert "Access is Denied" in response_data
+
+
+def test_trigger_dag_run_id(admin_client):
+    test_dag_id = "example_bash_operator"
+    test_run_id = "test_id"
+
+    admin_client.post(f'trigger?dag_id={test_dag_id}&run_id={test_run_id}')
+
+    with create_session() as session:
+        run = session.query(DagRun).filter(DagRun.dag_id == test_dag_id).first()
+    assert run is not None
+    assert run.run_type == DagRunType.MANUAL
+    assert run.run_id == test_run_id
```

Comment:
Do we have a test for calling `trigger` _without_ `run_id`? If so, the two should be combined with `pytest.mark.parametrize`.

Reply:
@uranusjr I didn't see a specific test for it, the other tests are for specific use cases like testing dag_params_conf and different tests for execution dates. I can make this test parameterized and check the outcome with or without run_id.

Reply:
It means at some point in the past this test was failing and we couldn’t figure out why. This should be the chance to un-quarantine it.

Reply:
got it. 
I removed the existing test from quarantine and parameterized it to check the use case of with and without run_id. 
Hopefully it will be stable now. thanks :relaxed: